packages/paste-website/src/components/icons-list/IconsList.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import {useUID, useUIDSeed} from 'react-uid';
+import {Box} from '@twilio-paste/box';
+import * as Icons from '@twilio-paste/icons/esm';
```

Comment:
So here I can now get all icons

Reply:
Not a fan of this. We're currently trying to work our way out of the barrel export files in `core-bundle` and it was an explicit choice not to include it for the icons package.  We can reasonably assume many more icons will be added to this package and the bundle size will grow quite a bit.  Folks can sometimes tend to import everything and never really go back to fix imports; instead they'll just say Paste is bloaty haha.

Can you import the JSON file and lazy load the icons instead? Something like (pseudocode)
```js
import {iconsJson} from 'icons/json/icons.json';

const IconComponents = [];
iconsJson.forEach(iconName => {
  iconComponents.push(require(`@twilio-paste/icons/cjs/${iconName}`);
});
```
Or maybe a dynamic import? `await import("@twilio-paste/icons/cjs/${iconName}")`